packages/LUIS/package.json

```diff
@@ -61,11 +61,11 @@
     "window-size": "^1.1.0"
   },
   "devDependencies": {
-    "typescript": "^3.1.1",
     "enum-files": "^1.1.0",
+    "mocha": "^5.2.0",
```

Comment:
Why the changes to the package.json? 

Can you revert these?

Reply:
I spent some time to get the tool to work, becaus it required a specific version of typescript. So, I though I might update the packages file to make everyone else's life easier going forward.

Reply:
That's great. Can you update the PR description to include the? Otherwise it looks like an accident.

Reply:
It's already in both title and description. Let me know if you think of a better wording